documentation/rosetta-modelling-component.md

```diff
@@ -1487,40 +1487,50 @@ An inline synonym can be expressed next to the attribute being mapped as follows
 E.g.
 
 ```
-type Collateral:
-    independentAmount IndependentAmount (1..1)
-        [synonym FpML_5_10 value "independentAmount"]
+type Engine:
+    engineSpecification EngineSpecification(1..1)
+        [synonym CONDITIONAL_SET_TO_EXAMPLE_1 value "engineDetail"]
 ```
 
 ##### External synonym
 
-External synonyms are defined inside the synonym source declaration so the synonym keyword and the synonym source are not required in every synonym. A synonym is added to an attribute by referencing the type and attribute name and then declaring the synonym to add as the synonym body enclosed in square brackets `[..]`. The code below removes all the synonyms from the `independentAmount` attribute of `Collateral` and then adds in a new synonym:
+External synonyms are defined inside the synonym source declaration so the synonym keyword and the synonym source are not required in every synonym. A synonym is added to an attribute by referencing the type and attribute name and then declaring the synonym to add as the synonym body enclosed in square brackets `[..]`. The code below removes all the synonyms from the `fuel` attribute of `EngineSpecification` as well as removing the synonyms for the `FuelEnum` for `Diesel` and `Gasoline`, so then adds in a new synonym:
 
 ```
-synonym source FpML_5_10 extends FpML
+synonym source EXTERNAL_SYNONYM_EXAMPLE_8 extends EXTERNAL_SYNONYM_EXAMPLE_8_BASE_2
 {
-    Collateral:
-        - independentAmount
-        + independentAmount
-            [value "independentAmount"]
+	EngineSpecification:
+		- fuel
+		+ fuel
+			[value "combustible"]
+
+	enums
```

Comment:
i think the EngineSpecification example is enough (no need to address the enum examples).

Reply:
i agree, only what is in EngineSpecification